src/configuration.js

```diff
@@ -112,6 +107,38 @@ export class Configuration {
 
     return rules;
   }
+
+  validate() {
+    const issues = [];
+
+    const defaultRuleNames = defaultRules.map(rule => rule.name);
+    var misConfiguredRuleNames = []
+      .concat(
+        this.options.only || [],
+        this.options.except || [],
+        this.options.rules || []
+      )
+      .map(toUpperCamelCase)
+      .filter(name => defaultRuleNames.indexOf(name) == -1);
+
+    if (this.getFormatter() == null) {
+      issues.push({
+        message: `The output format '${this.options.format}' is invalid`,
+        field: 'format',
+        type: 'error',
+      });
+    } else if (misConfiguredRuleNames.length > 0) {
```

Comment:
I'm thinking we should make this `if (...)` instead of `else if` this way both errors will show up.

Reply:
Oh yeah nice catch, what was my original intent 😄

Reply:
Added a spec to cover this use case